### PR TITLE
Chralu/issue28

### PR DESCRIPTION
**Pull request by Chralu**
# Fixes

- [x] Fix production builds
Thanks to @kaumudpa for detecting & fixing that issue.

- [x] Enable javascript chunks for a lighter load

- [ ] Shrink JS bundle size.

### CODE REVIEW

**Comment by kaumudpa** [comment]
@Chralu can we also please lock web3modal version @ 5.1.2?. We need to remove ^ as 5.1.3/4 is causing issues in graceful disconnect of connected wallet in android web.

Do try yourself on android chrome
Build with 5.1.3 - https://walletmodalts.web.app

Build with 5.1.4 - https://walletmodal1.web.app

Build with 5.1.2 - https://walletmodal3.web.app

File: README.md [hunk 1]
```diff
@@ -137,7 +137,9 @@ Following commands will build `JS` code to embed in the web browser.
 
 ```sh
 npm install
-npm run build_dev //or `npm run build` for production mode
+cd webpack
```
Comment (kaumudpa) [inline README.md:140]: No need of this now

File: package.json
Comment (kaumudpa) [inline package.json:1]: we need to lock this dep - `"@web3modal/wagmi": "5.1.1",` atleast for the time being. bugs in 5.1.3 & 5.1.4

**Comment by Chralu** [comment]
> @Chralu can we also please lock web3modal version @ 5.1.2?. We need to remove ^ as 5.1.3/4 is causing issues in graceful disconnect of connected wallet in android web.
> 
> Do try yourself on android chrome
> Build with 5.1.3 - https://walletmodalts.web.app
> 
> Build with 5.1.4 - https://walletmodalts1.web.app
> 
> Build with 5.1.2 - https://walletmodalts3.web.app - This works perfectly

I'll check this soon 👍

**Comment by Chralu** [comment]
@kaumudpa `@web3modal` 5.1.2 uses dependencies with highly critical issues.

I think we should create an upstream issue for disconnect issues on Android.
I create an issue on our repo as well to keep track of that.

**Pull request merged**